language/stdlib/modules/LibraConfig.move

```diff
@@ -73,6 +73,33 @@ module LibraConfig {
         reconfigure_();
     }
 
+    // Set a config item to a new value and trigger a reconfiguration.
+    public fun set_with_capability<Config: copyable>(
+        _cap: &ModifyConfigCapability<Config>,
+        payload: Config
+    ) acquires LibraConfig, Configuration {
+        let addr = CoreAddresses::LIBRA_ROOT_ADDRESS();
+        assert(exists<LibraConfig<Config>>(addr), ECONFIG_DOES_NOT_EXIST);
+        let config = borrow_global_mut<LibraConfig<Config>>(addr);
+        config.payload = payload;
+        reconfigure_();
+    }
+
+    // Publish a new config item. The caller will use the returned ModifyConfigCapability to specify the access control
+    // policy for who can modify the config.
+    public fun publish_new_config_with_capability<Config: copyable>(
+        config_account: &signer,
+        payload: Config,
+    ): ModifyConfigCapability<Config> {
+        assert(Roles::has_on_chain_config_privilege(config_account), ENO_CONFIG_PRIVILEGE);
+        assert(Signer::address_of(config_account) == CoreAddresses::LIBRA_ROOT_ADDRESS(), EINVALID_SINGLETON_ADDRESS);
+        move_to(config_account, LibraConfig { payload });
+        // We don't trigger reconfiguration here, instead we'll wait for all validators update the binary
```

Comment:
This  comment seems out of place given that not all `LibraConfig`s are validator-related.